config/locales/en.yml

```
@@ -2099,6 +2099,7 @@ en:
             course_subject_three:
               duplicate: *duplicate_course_subjects
             degrees_attributes:
+              duplicate: degree_attributes contain duplicate values
```

This is a weird English thing but I _think_ it should say 'contains' because `degree_attributes` is a single thing in the request body. Also, I think it's a bit clearer to say duplicate degrees, rather than values.
```suggestion
              duplicate: degree_attributes contains duplicate degrees
```

---

I think I made a typo, should be `degrees_attributes` in which case is plural

---

Good spot on the typo. I still think it's 'contains` because there is only one `degrees_attributes` key in the request body.